docs/zk-stack/running-a-hyperchain/locally.md

```diff
@@ -5,7 +5,13 @@ head:
       content: Deploying Locally | zkSync Docs
 ---
 
-# Deploying Locally
+# Getting Started with ZK Stack
+
+## Development dependencies
+
+Ensure you have followed <a href="https://github.com/matter-labs/zksync-era/blob/main/docs/guides/setup-dev.md" target="_blank">these instructions</a> to set up dependencies on your machine (don't worry about the Environment section for now).
```

Comment:
We should be consistent and use the square bracket notation (also helps with automatic dead links verification)
```suggestion
Ensure you have followed [these instructions](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/setup-dev.md) to set up dependencies on your machine (don't worry about the Environment section for now).
```

Reply:
Be sure to update all links on this page (I want to avoid adding duplicate comments in this PR)

Reply:
Updated all